### DISCUSSION TITLE
Thoughs on making `forge--prepare-post-buffer` insert more than one commit on PRs?

### DISCUSSION THREAD

**Discussion by iyefrat** [magit/forge#449] [Ideas]
Generally when I make pull requests, I would like the template to automatically insert all the commits to the post buffer, that way I have all the information relevant to the PR at hand, and can delete what I don't want (or just use as is). Is this type of thing something that you're interested in upstream? If so, I can submit a PR implementing it. Should it be behind a configuration variable?

**Comment by tarsius** [comment]
There's already a lot of technical dept surrounding the insert template/stub/... functionality, that I would like to repay before making any additions. So I would prefer if you implemented something for your own personal use instead.

**Reply by iyefrat** [reply]
Sure thing!
